### PR TITLE
Remove local gopath

### PR DESCRIPTION
**Pull request by adrianchiris**

### CODE REVIEW

**Comment by adrianchiris** [comment]
@SchSeba @zeeke  did some cleanup PTAL.

File: .github/workflows/buildtest.yaml [hunk 1]
```diff
@@ -38,9 +38,22 @@ jobs:
...
+    runs-on: ubuntu-22.04
+    steps:
+      - name: check out the repo
+        uses: actions/checkout@v3
+
+      - name: set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: build and test ib-sriov-cni image
+        run: make test-image
```
Comment (zeeke) [inline .github/workflows/buildtest.yaml:55]: nice!

**Review by zeeke — APPROVED** [review]
LGTM

**Review by SchSeba — APPROVED** [review]
LGTM

**Pull request merged**